webodf/lib/gui/SessionView.js

```diff
@@ -410,7 +410,7 @@ gui.SessionViewOptions = function () {
             var annotationViewManager = odfCanvas.getAnnotationViewManager();
             if (annotationViewManager) {
                 annotationViewManager.rehighlightAnnotations();
-                odtDocument.fixCursorPositions();
+                odtDocument.fixCursorPositions(); // TODO: remove this, this should not be needed. for now add return value if moved, and assert here that not.
```

Comment:
Not sure what highlighting this particular instance is intended for. Whilst you are correct that in an ideal world the whole fixCursorPositions function shouldn't be necessary to do this, it is definitely 100% required in the current imperfect world :smile: